database/migrations/2016_06_29_205018_add_oauth_fields_to_users.php

```diff
@@ -0,0 +1,45 @@
+<?php
```

Comment:
I don't think that auth for phoenix is available yet but this was added when integrating northstar-php and can be used later to authorize against northstar - is that correct @DFurnes ?